### DISCUSSION TITLE
are these logs normal?

### DISCUSSION THREAD

**Discussion by 0xMALVEE** [bbc/sqs-consumer#508] [Help] [answered]
### Summary

![image](https://github.com/bbc/sqs-consumer/assets/35369563/732b1523-9677-416c-9e28-f350693c98c7)

**Comment by nicholasgriffintn** [comment, marked as answer]
This indicates your application was unable to communicate with AWS SQS, you will need to check your configuration. We can't offer support for your application or communication with AWS services.
